src/verifact_manager.py

```
@@ -37,6 +37,7 @@
 class VerifactManager:
     def __init__(self, config: ManagerConfig = None):
         self.config = config or ManagerConfig()
+        self.claim_detector = ClaimDetector()
```

Ruff (0.11.9)

Undefined name `ClaimDetector`

(F821)